ban/http/schema.py

```diff
@@ -24,6 +24,15 @@
         'url': 'https://adresse.data.gouv.fr/api-gestion/',
     },
     'paths': {},
+    'tags': [
+        {'name': 'Municipality'},
+        {'name': 'Postcode'},
+        {'name': 'Group'},
+        {'name': 'Housenumber'},
+        {'name': 'Position'},
+        {'name': 'Diff'},
+        {'name': 'User'},
+    ],
```

Comment:
These Tags are only used in SwaggerUI.
They allow to group all routes by model.
![swaggerui_20161031_111700](https://cloud.githubusercontent.com/assets/12051203/19851212/03450744-9f5d-11e6-9be4-fcc32c343389.png)

Think it is interesting for user, don't you?